src/frontend/contexts/EarlyAccessContext.tsx

```diff
@@ -0,0 +1,75 @@
+import {createContext, useContext} from 'react';
+import {createStore, useStore, type StoreApi} from 'zustand';
+import {
+  createJSONStorage,
+  persist as createPersistedState,
+} from 'zustand/middleware';
+import {MMKVZustandStorage} from '../hooks/persistedState/createPersistedState';
+
+// NOTE: Do not change!
+export const STORAGE_KEY = 'EarlyAccessStore' as const;
+
+type EarlyAccessState = {
+  isEarlyAccessEnabled: boolean;
+};
```

Comment:
There is a high chance that we are going to change this (I think we are going to persist what is being set as early access). So if you use valibot, migrating will be much easier down the line